docs/README.md

```diff
@@ -17,6 +16,97 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+Quick Start
+===
+This documentation gives a quick start guide for running Spark with Celeborn.
```

Comment:
When a project belongs to ASF first appears in a artical, we should call it "Apache XYZ"

```suggestion
This documentation gives a quick start guide for running Apache Spark with Apache Celeborn(Incubating).
```